src/wccp/WccpConfig.cc

```diff
@@ -257,14 +186,6 @@ Svc_Ports_Too_Many(int line, int n)
   return ts::Errata::Message(14, LVL_INFO, out.str());
 }
 
-ts::Errata::Message
```

Comment:
Why is this removed? What if someone puts "Boo, Randal" as a service port value?

Reply:
The parser will catch it and will generate it's own standard messaging. And in the future, something better(canned yams?) will  catch it before it even happens (at validation).